app/forms/steps/capital/other_saving_type_form.rb

```diff
@@ -0,0 +1,11 @@
+module Steps
+  module Capital
+    class OtherSavingTypeForm < SavingTypeForm
+      private
+
+      def persist!
+        @saving = crime_application.savings.create(saving_type:)
+      end
```

Comment:
Note this differs to the savings type form behaviour. On the other saving type form, we will always create a new saving rather than directing a user to an existing incomplete saving for the saving type selected.